doc/classes/RibbonTrailMesh.xml

```diff
@@ -1,29 +1,39 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="RibbonTrailMesh" inherits="PrimitiveMesh" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
+		Represents a straight ribbon-shaped [PrimitiveMesh] with variable width.
 	</brief_description>
 	<description>
+		[RibbonTrailMesh] represents a straight ribbon-shaped mesh with variable width. The ribbon is composed of a number of flat or cross-shaped sections, each with the same [member section_length] and number of [member section_segments]. A [member curve] is sampled along the total length of the ribbon, meaning that the curve determines the size of the ribbon along its length.
 	</description>
 	<tutorials>
```

Comment:
Should link to https://docs.godotengine.org/en/latest/tutorials/3d/particles/trails.html and https://docs.godotengine.org/en/latest/tutorials/3d/particles/index.html